crates/bevy_asset/src/handle.rs

```diff
@@ -74,6 +74,13 @@ impl HandleId {
     }
 }
 
+/// A type for handler-related errors.
+#[derive(Debug)]
+pub enum HandleError {
+    /// Failed to convert HandleUntyped to Handle<T>.
```

Comment:
This should document what the stored string is.

Reply:
Is that what this decoration is for?

`#[error("foobar")]`